modules/auxiliary/gather/shodan_search.rb

```diff
@@ -36,6 +36,7 @@ def initialize(info = {})
       [
         OptString.new('SHODAN_APIKEY', [true, 'The SHODAN API key']),
         OptString.new('QUERY', [true, 'Keywords you want to search for']),
+        OptString.new('FACETS', [true, 'List of facets']),
```

Comment:
I assume we would want to have an option not to use `facets`, so maybe keep it as optional argument? 
```suggestion
        OptString.new('FACETS', [false, 'List of facets']),
```